CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+## Getting Started
+
+First, you'll need a couple things installed: Docker and asdf.
```

Comment:
Link to these tools?

Reply:
I knew I forgot something. I'll add links.